charts/templates/statefulset.yaml

```diff
@@ -21,6 +21,45 @@ spec:
       affinity: {{- toYaml . | nindent 8}}
       {{- end }}
       terminationGracePeriodSeconds: {{ .Values.terminationGracePeriod }}
+      {{- if ne .Values.bootstrapHostName "" }}
+      initContainers: 
+      - name: check-group-init-container
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        command:
+          - bash
+          - '-c'
+          - |
+            echo '### Begin init container ###'
```

Comment:
Begin Configuring Group. As there might be more than one init container in the future